tests/options-page/unit/formatters.test.js

```diff
@@ -0,0 +1,161 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I am not necessarily opposed to it, but why should this live here vs. in the svelte tests?